docs/source/notebooks/clv/clv-quickstart.ipynb

```diff
@@ -0,0 +1,2270 @@
+{
```

Comment:
<p>It would be nice to comment on the wide HDI for customer 6. Why is it so wide?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
<p>I think it's a natural feature of the BG/NBD + GG models. I added two short sentences on this after the example.</p>